rfcs/0098-community-team.md

```diff
@@ -0,0 +1,215 @@
+
+It is important for communities that are seeking to make the world better to have clear values, to articulate those values, and to be willing to stand up for them when necessary. Furthermore, since the Nix community is large and ideologically diverse, it is important that there be transparency as to what the community team is trying to accomplish. The preceding paragraphs are an attempt at finding this delicate balance.
+
+The statement of values is meant as a reminder of what we want to strive for, as a community. It's also a tool for community members to reference when discussing the direction of community efforts. Having the statement of values means that, for example, if a community member is suggesting a way to advance inclusion, nobody needs to argue as to whether inclusion should be in-scope for Nix, because the values make it clear that it is.
+
+To those who might oppose any public statement of values: Isn't it better to have it up front, so you don't have to wonder?
+
+To those who might say that any statement of values is merely performative and that the proof is in actions, not words: These authors fully agree, but we're saying the words anyway. Please hold us accountable for living up to them.
+
+## How new members join and leave the community team
+[joining-and-leaving]: #joining-and-leaving
+
+1. New members may apply to join the community team, and existing team members will decide on these applications. The team will evaluate applicants by the length of time they've been part of the Nix community, by their good character, by their demonstrated ability to exercise good judgement, and by their commitment to public service and a willingness to be self-critical. To form the initial team, the founding members will be appointed by Graham at his sole discretion.
+
+  The community team may decide to reach out to active community members they think would be a good fit for the team, if the member has not already applied to the team. The community team should keep internal records of what such outreach has been done so as to not duplicate efforts or reach out to members who have expressed disinterest prior.
+
+  The team's focus should be on developing leadership from within, not on bringing in experienced people from outside. This is important to maintaining ideological continuity.
+
+2. Community team members may resign at any time. Such resignations are effective immediately. In exceptional circumstances, members may also be removed by decision of the team; it is hoped that this will never be needed.
+
+3. The community team will decide on new members by consensus. That is, new members will be added to the team if and only if every then-current team member is in favor of adding them. In the event of removing members, the team will decide by a 2/3 majority vote, with the permissible votes being "yes" and "no". All team members are required to participate in membership decisions.
```

Comment:
*That is, new members will be added to the team if and only if every then-current team member is in favor of adding them. In the event of removing members, the team will decide by a 2/3 majority vote,*

Doesn't this effectively mean that only a 2/3rd of votes are needed to add someone to the team? A nefarious team could use the 2/3 majority vote to remove all members that do not agree and then accept the new member. They could then vote back in the 1/3 that they removed (if they are willing to).

While this does sound fairly theoretical, such politics can occur when a team becomes so small that 1/3 is only one or two persons.

I think this can be avoided by changing this to something like: N members or 2/3 of the members, whichever is the largest. (Which an appropriate value of N.)